src/graphnet/data/datamodule.py

```diff
@@ -72,6 +78,78 @@ def __init__(
 
         self.setup("fit")
 
+    def _set_dataloader_kwargs(
+        self,
+        train_dl_args: Dict[str, Any],
+        val_dl_args: Union[Dict[str, Any], None],
+        test_dl_args: Union[Dict[str, Any], None],
+    ) -> None:
+        """Copy train dataloader args to other dataloaders if not given.
+
+        Also checks that ParquetDataset dataloaders have multiprocessing
+        context set to "spawn" as this is strictly required.
+
+        See: https://docs.pola.rs/user-guide/misc/multiprocessing/
+        """
+        if val_dl_args is None:
+            self.info(
+                "No `val_dataloader_kwargs` given. This arg has "
+                "been set to `train_dataloader_kwargs` automatically."
+            )
+            val_dl_args = deepcopy(train_dl_args)
+        if (test_dl_args is None) & (self._test_selection is not None):
```

Comment:
Here `val_dl_args` and `test_dl_args` have a main difference with `train_dl_args`, which is the argument `suffle`.
 I understand that it has to be set to `False` for both of them and `True` for the trainins.
Is it not going to be a problem if no `args` are speified for the validation/test?